### DISCUSSION TITLE
Include a searchbar

### DISCUSSION THREAD

**Discussion by conradKE** [StanleyMasinde/events254Web-archive#108] [Ideas]
i think we should have a search  bar on the main page

we should also have a work-line timetable or timeline

**Comment by StanleyMasinde** [comment]
The search will be restored. It will be in the navigation bar so that it can be accessible on every page.

**Comment by conradKE** [comment]
nice one natumia moqups to strengthen the design ill share in a few

**Comment by StanleyMasinde** [comment]
Ok sure
